dts/bindings/saej2716/saej2716-controller.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: SAEJ2716 Single Edge Nibble Transmission (SENT) Controller
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+child-binding:
+  description: |
+    Each child node defines the configuration of a Single Edge Nibble Transmission (SENT) channel
+
+  properties:
+    reg:
+      type: array
+      required: true
+      description: Channel identifier.
+
+    num-data-nibbles:
+      type: int
+      enum:
+        - 1
+        - 2
+        - 3
+        - 4
+        - 5
+        - 6
+        - 7
+        - 8
+      required: true
+      description: |
+        Specifies the number of data nibbles. A nibble is a four-bit aggregation.
+        This property defines how many such nibbles are used in the data transmission.
+
+    tick-time-prescaler-us:
+      type: int
+      required: true
+      description: |
+        Specifies the tick time prescaler in microseconds (us). This value determines the
+        base time unit for the system's timing operations, effectively setting the resolution
+        of the timer.
+
+    calib-method-low-latency:
+      type: boolean
+      description: |
+        Indicates whether the successive calibration pulse check method with low latency is enabled.
+        This method ensures faster calibration by reducing the delay between successive pulses.
+
+    calib-pulse-range-25:
+      type: boolean
+      description: |
+        Indicates whether a 25% variation in the calibration pulse is acceptable.
+        If not, a 20% variation is acceptable.
+
+    pause-pulse-enable:
+      type: boolean
+      description: |
+        Indicates whether the channel is enabled to detect a pause pulse.
+
+    crc-check-disable:
```

Comment:
For CRC, as far as i understand:
- Fast Message: CRC is optional (can be enabled/disabled). If CRC is enabled, two main options:
   + Legacy 4-bit CRC
   + Recommended 4-bit CRC
- Short Serial Message: CRC seems always enabled, two options:
   + Legacy 4-bit CRC
   + Recommended 4-bit CRC
- Enhanced Serial Message: 6-bit CRC

Should we do something like:
- In dts bindings, create two properties for fast-crc and short-serial-crc
- Define macros for these options in a header file, which can be combined to configure the CRC

Reply:
I updated as your suggestion. added 2 properties `fast-crc` and `short-serial-crc`